process_scoresheets.py

```diff
@@ -74,28 +79,30 @@ def _validate_player_name(evaluator_name: str, row: Dict[str, Any]) -> str:
 
     return player_name
 
-def _validate_row(evaluator_name: str, row: Dict[str, Any]) -> Row:
+def _validate_row(evaluation_columns: List[str], min_score: int, max_score: int, evaluator_name: str, row: Dict[str, Any]) -> Row:
```

Comment:
It would be my preference to have a class/object that takes these params on instantiation so that they don't have to be passed around a zillion times. (I'll take this as a follow-up refactor.)